pubsub/kafkapubsub/go.sum

```diff
@@ -65,6 +65,8 @@ github.com/google/btree v0.0.0-20180813153112-4030bb1f1f0c/go.mod h1:lNA+9X1NB3Z
 github.com/google/go-cmp v0.2.0/go.mod h1:oXzfMopK8JAjlY9xF4vHSVASa0yLyX7SntLO5aqRK0M=
 github.com/google/go-cmp v0.3.0 h1:crn/baboCvb5fXaQ0IJ1SGTsTVrWpDsCWC8EGETZijY=
 github.com/google/go-cmp v0.3.0/go.mod h1:8QqcDgzrUqlUb/G2PQTWiueGozuR1884gddMywk6iLU=
+github.com/google/go-replayers/grpcreplay v0.1.0 h1:eNb1y9rZFmY4ax45uEEECSa8fsxGRU+8Bil52ASAwic=
```

Comment:
Why does `kafkapubsub` need `grpcreplay`? I guess this is just `go.sum`, but is this really needed? Same for `mongodocstore` and `natspubsub` and `gocdk` and `contributebot` and others.

Reply:
I'm not sure how to answer that. `go mod tidy` doesn't change anything.

For that matter, why does `docstore/mongodocstore` have aws entries in its `go.sum`?

I guess `go.sum` just accumulates stuff and never gets cleaned up. 

What action would you like me to take?

Reply:
I guess you are right, there is nothing that cleans `go.sum` up. Maybe we should delete them and recreate them from scratch periodically?

At any rate, maybe try deleting the new lines, run `go build && go test`; if they come back, fine, if they don't, let's avoid cluttering the file unnecessarily.

Reply:
I tried this with samples and runtimevar/etcd. In both cases the lines came back. I stopped experimenting at that point.